### PR TITLE
check if process is undefined

### PR DESCRIPTION
**Pull request by cryogenian**
It does not work in safari if without it

### CODE REVIEW

File: src/Signal/Time.purs [hunk 1]
```diff
@@ -34,7 +34,8 @@ every = runFn3 everyP constant now
 -- |Returns the number of milliseconds since an arbitrary, but constant, time in the past.
 foreign import now """
   function now() {
-    var perf = typeof performance !== 'undefined' ? performance : null;
+    var perf = typeof performance !== 'undefined' ? performance : null,
+        process = typeof process !== 'undefined' ? process : null;
```
Comment (michaelficarra) [inline src/Signal/Time.purs]: That's not going to work. Now `process` will always be `null`. Name it something else.

Comment (paf31) [inline src/Signal/Time.purs]: Does changing `process && ..` to `!!process && ..` not do the job?

Comment (michaelficarra) [inline src/Signal/Time.purs]: It's going to throw a `ReferenceError` if it's not a member of the global object.

Comment (bodil) [inline src/Signal/Time.purs]: So, (typeof process !== "undefined" && process.hrtime && ...) below, then? Odd behaviour, is this a JSCore thing?

Comment (cryogenian) [inline src/Signal/Time.purs]: <code>process</code> will not be <code>null</code> in nodejs, it's part of node's global object

Comment (cryogenian) [inline src/Signal/Time.purs]: changing <code>process</code> to <code>!!process</code> does not work because it is undefined in browsers

Comment (michaelficarra) [inline src/Signal/Time.purs]: @cryogenian: You are shadowing the global. Trust me, it will be `null`.

**Comment by cryogenian** [comment]
will it work now?

**Comment by michaelficarra** [comment]
Yeah, that's fine.

**Comment by bodil** [comment]
Looks good to me.

**Pull request merged**